attributes/defaults.rb

```diff
@@ -18,17 +18,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Attribute that defines whether MariaDB or MySQL should be used
-default['mysqld']['use_mariadb'] = true
+# Attribute that defines whether MariaDB, MySQL or Percona should be used
+default['mysqld']['db_install'] = 'mariadb'
```

Comment:
The only thing missing now is a note in the README explaining the use of the `db_install` attribute.